test/test_torch.py

```diff
@@ -2835,6 +2835,7 @@ def ri(indices):
 
         # Case 1: Purely Integer Array Indexing
         reference = conv_fn(consec((10,)))
+        self.assertEqual(reference[conv_fn(torch.LongTensor())], reference.new())  # empty tensor indexing
```

Comment:
I'm not sure if this test should go here. We aren't really testing advanced indexing here, merely `index_select`. Otherwise the code would have gone into a different path than `(getValue)`.

Reply:
Thanks. I'll check again. If I can't find its home, I'll put it into a new test.